src/common/ErrorBoundary.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class ErrorBoundary extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            hasError: false,
+        };
+    }
+
+    static getDerivedStateFromError() {
+        return {
+            hasError: true,
+        };
+    }
+
+    componentDidCatch(error, info) {
+        console.log(error);
+        console.log(info);
+    }
+
+    render() {
+        // Fallback UI Code
+        if (this.state.hasError) {
+            return <h1>Something Went Wrong</h1>;
```

Comment:
add button to reload the app (location.reload())

Reply:
Added the same